pyopensprinkler/station.py

```diff
@@ -48,13 +83,63 @@ def seconds_remaining(self):
         """Retrieve seconds remaining"""
         return self._get_status_variable(1)
 
+    @property
+    def start_time(self):
+        """Retrieve start time"""
+        return self._get_status_variable(2)
+
+    @property
+    def max_name_length(self):
+        return self._controller._state["stations"]["maxlen"]
+
+    @property
+    def masop(self):
+        return self._bit_check("masop")
+
+    @property
+    def masop2(self):
+        return self._bit_check("masop2")
+
+    @property
+    def ignore_rain(self):
+        return self._bit_check("ignore_rain")
+
+    @property
+    def ignore_sn1(self):
+        return self._bit_check("ignore_sn1")
+
+    @property
+    def ignore_sn2(self):
+        return self._bit_check("ignore_sn2")
+
+    @property
+    def stn_dis(self):
+        return self._bit_check("stn_dis")
+
+    @property
+    def stn_seq(self):
+        return self._bit_check("stn_seq")
+
+    @property
+    def stn_spe(self):
```

Comment:
I couldn't figure out what this refers to lol ... maybe `special` 🤷

Reply:
It is special I believe. If the bit is set you can get extra info from the `/je` endpoint from what I can tell in the doc. ie: GPIO or HTTP type stations etc. Renamed to `special`.